core/src/main/scala/com/raphtory/api/analysis/graphview/DeployedTemporalGraph.scala

```diff
@@ -3,41 +3,35 @@ package com.raphtory.api.analysis.graphview
 import cats.effect.IO
 import cats.effect.unsafe.implicits.global
 import com.raphtory.internals.components.querymanager.Query
-import com.raphtory.internals.graph.GraphAlteration
 import com.raphtory.internals.management.QuerySender
 import com.typesafe.config.Config
 
 /** Root class for local deployments of the analysis API
   *
   * A DeployedTemporalGraph is a [[TemporalGraph]]
   * with a [[Deployment]] object attached to it that allows it to stop.
-  * To create a `DeployedTemporalGraph` use the `stream` or `load` method
-  * of the [[com.raphtory.Raphtory Raphtory]] object.
   *
   * @param deployment The deployment object used to control Raphtory
   *
-  * @see [[TemporalGraph]], [[Deployment]], [[com.raphtory.Raphtory Raphtory]]
+  * @see [[TemporalGraph]], [[Deployment]]
   */
 class DeployedTemporalGraph private[raphtory] (
     override private[api] val query: Query,
     override private[api] val querySender: QuerySender,
     override private[api] val conf: Config,
-    private[api] val local: Boolean,
     private val shutdown: IO[Unit]
 ) extends TemporalGraph(query, querySender, conf)
         with AutoCloseable {
 
   def config: Config = conf
 
   def destroy(force: Boolean = false): Unit = {
-    if (!local)
-      querySender.destroyGraph(force)
+    querySender.destroyGraph(force)
     shutdown.unsafeRunSync()
   }
 
   override def close(): Unit = {
```

Comment:
We shouldn't expose a `close` method to the user anymore. We need to ensure that we are calling `querySender.disconnect()` at the end of `runWithgraph`. As for the `shutdown` hook, this is set to `IO.unit` in all the cases now, so we can remote it and stop using it inside `destroy`

Reply:
Fixed